bin/sql-dump-to-entities.js

```diff
@@ -0,0 +1,188 @@
+const _ = require('lodash')
+const fs = require('fs')
+const path = require('path')
+const colors = require('colors')
```

Comment:
afaict you don't use this anymore.

Reply:
oh thanks :) yeah I did some more logging initially of the differences but resolved all those and didn't need it anymore